crates/bevy_pbr/src/render/pbr_prepass.wgsl

```diff
@@ -53,6 +53,7 @@ fn fragment(
 #ifdef VERTEX_TANGENTS
 #ifdef STANDARD_MATERIAL_NORMAL_MAP
 
+// TODO: Transforming UVs mean we need to apply derivative chain rule for meshlet mesh material pass
```

Comment:
Just for clarification for my understanding, this was also the case before this PR correct?

Reply:
Yes. I'll need to do it as some point, low priority atm.